src/components/CustomFileInput/style.scss

```diff
@@ -15,3 +15,7 @@
   width: 100%;
   z-index: 1;
 }
+
+.uploadCopy {
+  display: block;
+}
```

Comment:
I think this is no longer needed for this component since there is a `display: block;` at the `Uploader` parent container level.

Reply:
You are correct.